eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeMojoTest.java

```diff
@@ -31,23 +32,30 @@
 import org.eolang.xax.XaxStory;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
 import org.junit.jupiter.params.ParameterizedTest;
 
 /**
  * Test case for {@link BinarizeMojo}.
  *
  * @since 0.1
  */
+@Execution(ExecutionMode.CONCURRENT)
 final class BinarizeMojoTest {
 
     @Test
-    void binarizesSimpleEoProgram(@TempDir final Path temp) throws Exception {
+    void parsesSimpleEoProgram(@TempDir final Path temp) throws Exception {
```

Comment:
@levBagryansky Compound name is too heavy, maybe we can cal it just `parse`? According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html